### ISSUE TITLE
Upon selecting Next page/Previous page, Narrator remains silent

### ISSUE THREAD

**Issue by ayushjai19** [microsoft/WinUI-Gallery#2086]
### Which version of the app?

WinUI 3 Gallery

### Description

Environment Details:
OS Version: 24H2 (OS Build 26100.4315)
App: WinUI 3 Gallery
App Version: 2.6.0.0 (SDK 1.7)
AT: Color Contrast

Repro Steps:
Launch the WinUI 3 Gallery app.
Navigate to search box using tab key.
Type 'Pipspager' in search box and activate it. 
Now navigate to 'Next/Previous page' button under 'Pipspager with options to change its orientation and button visibility'. And activate it. Observe Narrator announcement.
Actual Result:
Upon selecting Next page/Previous page, Narrator remains silent Instead of announcing the appeared page information and focused element information.

Expected Result:
Upon selecting Next page/Previous page, Narrator should not remain silent, Instead it should announce the selected page Like, Page 5 selected.

User impact:
Users who rely on Screen reader will face difficulty in understanding the updated page information.

MAS Reference Link: [MAS 4.2.12 – Change Notification](https://aka.ms/MAS4.2.12)

Trap ID: [1.9 - Feedback Failure](https://aka.ms/FeedbackFailure)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

### Windows version

Windows 11 24H2 (26100)
